src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.agora.server.business
+
+import org.broadinstitute.dsde.agora.server.AgoraConfig
+import org.broadinstitute.dsde.agora.server.dataaccess.AgoraDao
+import org.broadinstitute.dsde.agora.server.model.AgoraEntity
+
+/**
+ * Created by dshiga on 5/13/15.
+ */
+object AgoraBusiness extends AgoraDao {
+
```

Comment:
We shouldn't need to extend AgoraDao here.. you should be able to import AgoraDao._ and have access to the factory methods.